galleries/examples/color/color_sequences.py

```diff
@@ -0,0 +1,66 @@
+"""
+=====================
+Named Color Sequences
```

Comment:
```suggestion
Named color sequences
```
Per our capitalization policy